features/build/stibuild.feature

```diff
@@ -4,7 +4,7 @@ Feature: stibuild.feature
     Given I have a project
     Given I obtain test data file "templates/<template>"
     And I run the :new_app client command with:
-      | file | <template> |
+      | file | tc498848-s2i.json |
```

Comment:
this change doesn't look right

Other than that looks good.

Reply:
The original snenario is a multiple one, now other cases have removed from 4.x, only this case is left. I don't want to change the automation field in polarion , just update this case instead

Reply:
but few lines above you have `I obtain test data file "templates/<template>"` which depends on the parameter. It doesn't make sense to me. Updating polarion is just calling a simple script 
https://mojo.redhat.com/docs/DOC-1125811

Although I don't see why you'd need to change.

Reply:
Ok, I've got it. Not ideal but anyway.